packages/api/portfolioDrafts/submit/submitPortfolioDraft.ts

```diff
@@ -33,6 +36,14 @@ export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayPr
 
   try {
     const result = await submitPortfolioDraftCommand(TABLE_NAME, portfolioDraftId);
+    const data = await sqsClient.send(
+      new SendMessageCommand({
+        QueueUrl: QUEUE_URL,
+        MessageBody: JSON.stringify(result.Attributes as PortfolioDraft),
+        MessageGroupId: "submitPortfolioDraft",
+        MessageDeduplicationId: result?.Attributes?.submit_id,
```

Comment:
i don't like that this could be `undefined`. If we don't have a `result` and if `result` doesn't have `Attributes` we should crash loudly

Reply:
Since I am switching to a normal queue, from a FIFO queue, we can just delete the deduplication